tests/run-make/extra-filename-with-temp-outputs/rmake.rs

```diff
@@ -0,0 +1,23 @@
+// In order to prevent temporary files from overwriting each other in parallel
+// compilation, rustc was changed to mix an extra filename with temporary
+// outputs. However, as this is a similar behavior with the codegen flag
+// -C extra-filename, this test checks that the manually passed flag
+// is not overwritten by this feature, and that the output files
+// are named as expected.
+// See https://github.com/rust-lang/rust/pull/15686
+
+//FIXME(Oneirical): ignore-cross-compile
```

Comment:
Suggestion: remove this FIXME if try job comes back green since it seems to work

Reply:
I think this is not yet addressed.